### ISSUE TITLE
In parallel mode, tox marks skipped interpreters as succeded.

### ISSUE THREAD

**Issue by awecx** [tox-dev/tox#1495]
Say I have no Python3.3 interpreter on my machine.
The following tox.ini:
```
[tox]
envlist = py33
skip_missing_interpreters = True
```

would of course yield: 

```
python3 -m tox
GLOB sdist-make: /xxx/yyy/setup.py
py33 recreate: /xxx/yyy/.tox/py33
SKIPPED: InterpreterNotFound: python3.3
__________________________________________________________________ summary ___________________________________________________________________
SKIPPED:  py33: InterpreterNotFound: python3.3
```
but when run in parallel, we get:
```
python3 -m tox -p auto
GLOB sdist-make: /xxx/yyy/setup.py
✔ OK py33 in 0.22 seconds
__________________________________________________________________ summary ___________________________________________________________________
  py33: commands succeeded
  congratulations :)
```

That could make a user to think that his tests are green while they simply … did not run.
Tested with tox 3.14.3.

**Comment by rpkilby** [comment]
Duplicate of #1274.
